### DISCUSSION TITLE
Release a new version

### DISCUSSION THREAD

**Discussion by sandrotosi** [transmission/transmission#2329] [General]
3.00 was released on May 22, 2020, more an a year and a half ago, and there have been 573 commits on master since then.

Please release a new version of transmission.

Thanks!

**Comment by mikedld** [comment]
The closest you can get: https://github.com/transmission/transmission/issues/1528#issuecomment-990585316

**Comment by ckerr** [comment]
There are a couple of big things that need to be written & land before the next version, but it will happen.

Just thinking out loud -- we may want to number the next version as 4 due to the extensive rewriting.

**Comment by ckerr** [comment]
There are now three separate "new release" discussions which seems counterproductive, so I'm going to lock two of them :)
